superset/base/kustomization.yaml

```diff
@@ -25,143 +24,94 @@ configMapGenerator:
     fieldref:
-      fieldpath: data.SUPERSET_VOLUME_SIZE
-  - name: SUPERSET_SECRET_KEY
+      fieldpath: data.superset_cpu_limits
+  - name: superset_db_memory_requests
     objref:
       name: superset-config
       kind: ConfigMap
       apiVersion: v1
     fieldref:
-      fieldpath: data.SUPERSET_SECRET_KEY
-  - name: SUPERSET_DB_USERNAME
+      fieldpath: data.superset_db_memory_requests
+  - name: superset_db_memory_limits
     objref:
       name: superset-config
       kind: ConfigMap
       apiVersion: v1
     fieldref:
-      fieldpath: data.SUPERSET_DB_USERNAME
-  - name: SUPERSET_DB_PASSWORD
+      fieldpath: data.superset_db_memory_limits
+  - name: superset_db_cpu_requests
     objref:
       name: superset-config
       kind: ConfigMap
       apiVersion: v1
     fieldref:
-      fieldpath: data.SUPERSET_DB_PASSWORD
-  - name: LOG_LEVEL
+      fieldpath: data.superset_db_cpu_requests
+  - name: superset_db_cpu_limits
     objref:
       name: superset-config
       kind: ConfigMap
       apiVersion: v1
     fieldref:
-      fieldpath: data.LOG_LEVEL
-  - name: SUPERSET_MEMORY_REQUESTS
+      fieldpath: data.superset_db_cpu_limits
+  - name: storage_class
     objref:
       name: superset-config
       kind: ConfigMap
       apiVersion: v1
     fieldref:
-      fieldpath: data.SUPERSET_MEMORY_REQUESTS
-  - name: SUPERSET_MEMORY_LIMITS
-    objref:
-      name: superset-config
-      kind: ConfigMap
-      apiVersion: v1
-    fieldref:
-      fieldpath: data.SUPERSET_MEMORY_LIMITS
-  - name: SUPERSET_CPU_REQUESTS
-    objref:
-      name: superset-config
-      kind: ConfigMap
-      apiVersion: v1
-    fieldref:
-      fieldpath: data.SUPERSET_CPU_REQUESTS
-  - name: SUPERSET_CPU_LIMITS
-    objref:
-      name: superset-config
-      kind: ConfigMap
-      apiVersion: v1
-    fieldref:
-      fieldpath: data.SUPERSET_CPU_LIMITS
-  - name: SUPERSET_DB_MEMORY_REQUESTS
-    objref:
-      name: superset-config
-      kind: ConfigMap
-      apiVersion: v1
-    fieldref:
-      fieldpath: data.SUPERSET_DB_MEMORY_REQUESTS
-  - name: SUPERSET_DB_MEMORY_LIMITS
-    objref:
-      name: superset-config
-      kind: ConfigMap
-      apiVersion: v1
-    fieldref:
-      fieldpath: data.SUPERSET_DB_MEMORY_LIMITS
-  - name: SUPERSET_DB_CPU_REQUESTS
-    objref:
-      name: superset-config
-      kind: ConfigMap
-      apiVersion: v1
-    fieldref:
-      fieldpath: data.SUPERSET_DB_CPU_REQUESTS
-  - name: SUPERSET_DB_CPU_LIMITS
-    objref:
-      name: superset-config
-      kind: ConfigMap
-      apiVersion: v1
-    fieldref:
-      fieldpath: data.SUPERSET_DB_CPU_LIMITS
-  - name: SUPERSET_IMAGE
-    objref:
-      name: superset-config
-      kind: ConfigMap
-      apiVersion: v1
-    fieldref:
-      fieldpath: data.SUPERSET_IMAGE
-
+      fieldpath: data.storage_class
 
 configurations:
   - params.yaml
 
 generatorOptions:
   disableNameSuffixHash: true
+
+images:
+  - name: superset
+    newName: quay.io/internaldatahub/superset
```

Comment:
Isn't there a different image we want to use here?
@maulikjs @4n4nd

Reply:
Other relevant PR: https://github.com/opendatahub-io/odh-manifests/pull/454

Reply:
I changed to `quay.io/opendatahub/superset:1.1.0`